app/forms/instance-create.tsx

```diff
@@ -318,7 +336,11 @@ export function CreateInstanceForm() {
               />
```

Comment:
You should disable the tab triggers for silo/project images like you did on the instance size ones.